src/app/core/services/user.service.ts

```diff
@@ -0,0 +1,86 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/observable/throw';
+import 'rxjs/add/observable/of';
+import { User } from '../models/user';
+
+export const MOCK_USER = new User();
+MOCK_USER._id = '1';
+MOCK_USER.email = '1';
+MOCK_USER.firstName = 'Foo';
+MOCK_USER.lastName = 'Bar';
+MOCK_USER.password = '1';
+
+/**
+ * The user service.
+ */
+@Injectable()
+export class UserService {
+
+    /**
+     * True if authenticated
+     * @type
+     */
+    private _authenticated = false;
```

Comment:
you don't need to prefix private properties with underscore - you have typescript!

Reply:
Yes, but i have function with the same name.